clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/UpsertAmazonLoadBalancerClassicDescription.java

```diff
@@ -176,5 +178,26 @@ public String getSslCertificateId() {
     public void setSslCertificateId(String sslCertificateId) {
       this.sslCertificateId = sslCertificateId;
     }
+
+    public List<CookiePolicy> getCookiePolicies() {
+      return cookiePolicies;
+    }
+
+    public void setCookiePolicies(List<CookiePolicy> cookiePolicies) {
+      this.cookiePolicies = cookiePolicies;
+    }
+
+    @Data
+    public static class CookiePolicy {
+      public enum CookieType {
+        APP,
+        LB
+      }
+
+      public CookieType cookieType;
+      public String policyName;
+      public String cookieName;
+      public long durationSeconds;
```

Comment:
`durationSeconds` available for `LBCookieStickinessPolicy` or  `CookieType.LB`

Reply:
This PR comment should definitely be a comment on the property itself.